### PR TITLE
Add additional tests for Plan::isSamePerson

### LINKED ISSUE #69: Add testcases for Plan::isSamePlan

### PR DESCRIPTION
**Pull request by Elijah5399**
Closes #69

### CODE REVIEW

File: src/test/java/seedu/address/model/plan/PlanTest.java [hunk 1]
```diff
@@ -46,6 +46,33 @@ public void equals() {
         assertFalse(BALI.equals(editedBaliPlan));
     }
 
+    @Test
+    public void isSamePlan() {
```
Comment (GohTengFong) [inline src/test/java/seedu/address/model/plan/PlanTest.java:50]: I love how you added comments for every assert statement! Makes it clear and understandable.

File: src/test/java/seedu/address/model/plan/PlanTest.java [hunk 2]
```diff
@@ -46,6 +46,33 @@ public void equals() {
...
+        editedBali = new PlanBuilder(BALI).withPlanName("Dota 2").build();
+        assertFalse(BALI.isSamePlan(editedBali));
+
+        // name differs in case, all other attributes same -> returns false
+        editedBali = new PlanBuilder(BALI).withPlanName("Bali Trip".toLowerCase()).build();
+        assertFalse(BALI.isSamePlan(editedBali));
+
+        // name has trailing spaces, all other attributes same -> returns false
+        String planNameWithTrailingSpaces = "Bali Trip" + " ";
+        editedBali = new PlanBuilder(BALI).withPlanName(planNameWithTrailingSpaces).build();
```
Comment (GohTengFong) [inline src/test/java/seedu/address/model/plan/PlanTest.java:72]: I appreciate the easy naming of the functions and variables which allows me to treat them like blackboxes but still understand what each specific test does!

**Review by GohTengFong — APPROVED** [review]

**Comment by GohTengFong** [comment]
There is a fundamental understanding of commenting, utilising of testcases as well as variable and function naming at play here. This PR demonstrates that pinnacle of Software Engineering. As a reviewer, I am grateful for the opportunity to review such a masterpiece and I hope I would be able to incorporate such genius in my own work. LGTM. LFG.

File: src/test/java/seedu/address/model/plan/PlanTest.java [hunk 3]
```diff
@@ -46,6 +46,33 @@ public void equals() {
...
+        assertFalse(BALI.isSamePlan(editedBali));
```
Comment (victorlaiyeeteng) [inline src/test/java/seedu/address/model/plan/PlanTest.java:73]: Simple yet well-described explanation for each test!

**Review by victorlaiyeeteng — APPROVED** [review]
This test of isSamePlan() is more comprehensive! LGTM

**Pull request merged**